src/jsolve.jl

```diff
@@ -0,0 +1,20 @@
+#this file contains functions/wrappers related to the solve function in FEniCS
+#https://fenicsproject.org/olddocs/dolfin/1.3.0/python/programmers-reference/fem/solving/solve.html
+using FEniCS
+lvsolve(a,L,u)=fenics.solve(a.pyobject==L.pyobject, u.pyobject)
+lvsolve(a,L,u,bcs)=fenics.solve(a.pyobject==L.pyobject, u.pyobject, bcs=bcs.pyobject)
+export lvsolve
+
+nlvsolve(F,u;bcs=nothing)=fenics.solve(F.pyobject==0,u.pyobject,bcs=bcs.pyobject)
+export nlvsolve
+
+anlvsolve(F,a,u,bcs,tol,M)=fenics.solve(F.pyobject==a.pyobject,u.pyobject,bcs=bcs.pyobject,tol=tol,M=M)
```

Comment:
FEniCS provides different types of solve (https://fenicsproject.org/olddocs/dolfin/1.3.0/python/programmers-reference/fem/solving/solve.html).
the anlvsolve is the adaptive non-linear solver. I should probably document these more clearly.